lib/Importer.js

```diff
@@ -265,6 +272,11 @@ export default class Importer {
 
           newImports.push(jsModule.toImportStatement(this.config));
         });
+        // The "push" method on the ImportStatements class only accepts one ImportStatement
+        // at a time. Hence the need to loop.
```

Comment:
We should change the push method to accept multiple `ImportStatement`s at a time. I'm happy to add a commit to this PR that does that if you like.